src/workerd/io/worker-interface.capnp

```diff
@@ -482,6 +488,16 @@ struct JsValue {
   }
 }
 
+interface AbortSignal $Cxx.allowCancellation {
+    struct Params {
+      reason @0 :JsValue;
+    }
+
+    struct Results {}
+
+    triggerAbort @0 Params -> Results;
```

Comment:
Why are `Params` and `Results` being factored out as separate structs here? You can just write:

```suggestion
    triggerAbort @0 (reason :JsValue) -> ();
```

This is exactly equivalent -- in fact, it's binary-compatible, so you could change it to structs later if desired.